From a92c9e0dfda14ed6c82b46f2ecd76dc9ea0734f3 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Thu, 1 Nov 2018 12:12:54 +0100
Subject: [PATCH] pmem: fix build error on powerpc.

References: FATE#326628, bsc#1113295
Patch-mainline: no, not needed

Fix drivers/nvdimm/pmem.c:23:10: fatal error: asm/set_memory.h: No such file or
directory

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/include/asm/Kbuild | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/powerpc/include/asm/Kbuild b/arch/powerpc/include/asm/Kbuild
index 5c4fbc80dc6c..f6ab40874caf 100644
--- a/arch/powerpc/include/asm/Kbuild
+++ b/arch/powerpc/include/asm/Kbuild
@@ -7,4 +7,5 @@ generic-y += local64.h
 generic-y += mcs_spinlock.h
 generic-y += preempt.h
 generic-y += rwsem.h
+generic-y += set_memory.h
 generic-y += vtime.h
